<?xml version="1.0"?>
<launch>
	<!-- Ros topics -->
	<param name="/planning_cleaner/waypoints_topic" value="/planning/waypoints"/>
	<param name="/planning_cleaner/clean_waypoints_topic" value="/planning/waypoints_clean"/>

	<!-- Simple waypoints cleaner parameters -->
	<param name="/planning_cleaner/cleaner_frame_id" value="map"/>
	<param name="/planning_cleaner/n_path_to_keep" value="10"/>
	<param name="/planning_cleaner/radius_to_merge_path" value="2"/>
	<param name="/planning_cleaner/min_waypoint_occur" value="3"/>

	<node name="waypoints_cleaner_node" pkg="waypoint_cleaner" type="simple_cleaner_node.py" output="screen"/>
</launch>
